doc/user/layouts/shortcodes/private-preview.html

```diff
@@ -5,7 +5,7 @@
   It is under active development and may have stability or performance issues.
   It isn't subject to our backwards compatibility guarantees.
   {{if not (.Get "enabled-by-default")}}
-  <br><br>
+  <br>
```

Comment:
minor, but the extra line break here was intentional; it all just kind of runs together otherwise at certain screen sizes.

![image](https://github.com/user-attachments/assets/83714d8d-609a-40b9-9b80-7b04064cebba)